deploy/operations/monitoring.md

```diff
@@ -46,3 +46,338 @@ You will need to change the values in the `prometheus` fields in your metadata t
+
+### Check detection capabilities
+
+<table>
+    <tr>
+        <th rowspan="2">Check (+readiness)</th>
+        <th colspan="12">Failure</th>
+    </tr>
+    <tr>
+        <th>1</th>
+        <th>2</th>
+        <th>3</th>
+        <th>4</th>
+        <th>5</th>
+        <th>6</th>
+        <th>7</th>
+        <th>8</th>
+        <th>9</th>
+        <th>10</th>
+        <th>11</th>
+        <th>12</th>
+    </tr>
+    <tr>
+        <td>🚀 /healthy</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+    </tr>
+    <tr>
+        <td>🛠️ Normal usage metrics</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>🔶</td>
+        <td>🔶</td>
+    </tr>
+    <tr>
+        <td>✅ DAR identity</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>🔶↓↓</td>
+    </tr>
+    <tr>
+        <td>🚧 Per-USS heartbeat</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>🔶</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>🔶↓</td>
+    </tr>
+    <tr>
+        <td>🚧 Nonce exchange</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>🔶↓</td>
+    </tr>
+    <tr>
+        <td>🚀 DSS entity injection</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>✅</td>
+        <td>🔶</td>
+        <td>🔶</td>
+    </tr>
+    <tr>
+        <td>🛠️ Database metrics</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>❌</td>
+        <td>🔶</td>
+        <td>❌</td>
+        <td>✅</td>
+        <td>✅</td>
+    </tr>
+</table>
+
+#### Legend
+
+<table>
+    <tr>
+        <td rowspan="4">Readiness</td>
+        <td>🚀</td><td>Released</td>
+        <td rowspan="4">Failure detection</td>
+        <td>✅</td><td>Detects failure</td>
+    </tr>
+    <tr>
+        <td>✅</td><td>Complete (not yet released)</td>
+        <td>🔶</td><td>May detect failure</td>
+    </tr>
+    <tr>
+        <td>🚧</td><td>Not complete</td>
+        <td>🔶↓</td><td>Might possibly detect failure</td>
```

Comment:
Not sure I understand the difference between the two 🔶?

Reply:
I'm trying to convey qualitative likelihood -- 🔶 is something like "sometimes", 🔶↓ is something like "rarely", and 🔶↓↓ is something like "very rarely".  I'll see if I can improve the wording.